src/controller/python/chip/yaml/runner.py

```diff
@@ -299,6 +425,22 @@ def _attribute_read_action_factory(self, test_step, cluster: str):
         except ParsingError:
             return None
 
+    def _attribute_subscribe_action_factory(self, test_step, cluster: str):
+        '''Creates subscribe attribute command from TestStep provided.
+
+        Args:
+          'test_step': Step containing information required to run subscribe attribute action.
+          'cluster': Name of cluster write attribute action is targeting.
+        Returns:
+          SubscribeAttributeAction if 'test_step' is a valid subscribe attribute to be executed.
+          None if we were unable to use the provided 'test_step' for a known reason that is not
+          fatal to test execution.
+        '''
+        try:
+            return SubscribeAttributeAction(test_step, cluster, self._context)
+        except ParsingError:
+            return None
```

Comment:
Some comment about what parsing error we can get and why we do not let exceptions propagate would be useful here.